api/lib/logger.js

```diff
@@ -8,17 +8,10 @@ var logger = new winston.Logger({
       prettyPrint: true,
       colorize: true,
       timestamp: true,
-      handleExceptions: false,
-      logLevel: 'info',
-      showLevel: true,
-      silent: config.get('NODE_ENV') === 'test'
-    })
-  ],
-  exceptionHandlers: [
-    new winston.transports.Console({
-      prettyPrint: true,
-      colorize: true,
-      timestamp: true
+      handleExceptions: true,
```

Comment:
I think winston does not propagate the exception if this is set. So this means we wouldn't see the stack trace for uncaught exceptions when running in test mode.